developer/reference/articles/tooling/blade-cli/08-extending-blade-cli/03-creating-custom-project-templates-for-blade-cli.markdown

```diff
@@ -0,0 +1,80 @@
+---
+header-id: creating-custom-project-templates-for-blade-cli
+---
+
+# Creating Custom Project Templates for Blade CLI
+
+[TOC levels=1-4]
+
+Blade comes with over 32+ project templates, but many times you may feel that
+those are too simple or don't fit the need for your development team. You can
+create new custom project templates that fit your team's workflow and have Blade
+use it instead.
+
+The setup of a custom project template should mimic that of a Maven archetype.
+The best way to illustrate this is by visualizing a
+[sample template](https://github.com/liferay/liferay-blade-cli/tree/master/extensions/sample-template)'s
+folder structure:
+
+- `src/`
+    - `main/resources/`
+        - `META-INF`
+            - `maven`
+                - `archetype-metadata.xml`
+            - `archetype-post-generate.groovy`
+        - `archetype-resources`
+            - Folder structure to be generated
```

Comment:
I'm only backticking the folders/files. Since this is just a summary of what's inside the `archetype-resources` folder, it distinguishes it as a summary rather than an actual name.

Reply:
Understood :+1: